### PR TITLE
Add/ Remove ToDo PR

### PR DESCRIPTION
**Pull request by Javed-Shahbaz**
- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).

### CODE REVIEW

**Review by Reem-lab — COMMENTED** [review]
Hi @Javed-Shahbaz, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have :

- [ ]  I noticed you are missing key features section so please add 1 to 3 key features for this project and future features section since it is very important to add them to have a professional README 🔥 kindly do the same for the front end repo 🙏🏻✨

- [ ] kindly update the back to top link to be in the side as the given template 🤩🔥

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Reem-lab — COMMENTED** [review]
Hi @Javed-Shahbaz, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have :

- [ ]  I noticed you are missing futurefeatures section so please add 1 to 3 futurefeatures for this project and `future features section` since it is very important to add them to have a professional README 🔥 kindly do the same for the front end repo 🙏🏻✨ I mentioned this in the first review

- [ ] kindly update the back to top link to be in the side as the given template 🤩🔥

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -128,42 +209,92 @@ Example:
...
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
-<p allign="right">(<a href="#readme-top">Back to Top</a>)</p>
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (elisha2kyakpo1) [inline README.md:299]: - [ ] [OPTIONAL] This part of can be removed because it is just instructions for generating licenses.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Javed-Shahbaz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

👍 The app works as expected (delete, edit, local store, and status of a task.)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by didierganthier — APPROVED** [review]
Hi @Javed-Shahbaz ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Review by zieeco — COMMENTED** [review]
Hey @Javed-Shahbaz, According to this [review summary](https://github.com/Javed-Shahbaz/ToDoProject/pull/3#pullrequestreview-1368023559), Your project's being approved.

Cheer & Happy Coding!

**Pull request merged**